File: calc.cpp
#include <iostream>
using namespace std;
int main() {
    int operandoUno, operandoDos, opcion, resultado;
    cout << "Ingrese primer valor: ";
    cin >> operandoUno;
    cout << "Ingrese segundo valor: ";
    cin >> operandoDos;
    cout << "1. Suma" << endl;
    cout << "2. Resta\n";
    cout << "3. Multiplicacion\n";
    cout << "4. Division\n";
    cout << "Elija su opcion: ";
    cin >> opcion;
    switch (opcion)
    {
    case 1:
        resultado = operandoUno + operandoDos;
        break;
    case 2:
        resultado = operandoUno - operandoDos;
        break;
    case 3:
        resultado = operandoUno * operandoDos;
        break;
        case 4:
        resultado = operandoUno / operandoDos;
        break;
    default:
        cout << "~_~ un cursito de comprension lectora no te caeria mal";
        break;
    }
    cout << "El resultado es: " << resultado << endl;
    return 0;
}

File: calc.py
operandoUno = int(input('Ingrese primer valor: '))
operandoDos = int(input('Ingrese segundo valor: '))
print('1. Suma')
print('2. Resta')
print('3. Multiplicacion')
print('4. Division')
opcion = int(input('Elija su opcion: '))
resultado = 0
if opcion == 1:
    resultado = operandoUno + operandoDos
if opcion == 2:
    resultado = operandoUno - operandoDos
if opcion == 3:
    resultado = operandoUno * operandoDos
if opcion == 4:
    resultado = operandoUno / operandoDos
print('El resultado es: ', resultado)